bin/docker.js

```diff
@@ -0,0 +1,95 @@
+#! /usr/bin/env node
+/* eslint-disable no-console */
+const util = require('util'); // eslint-disable-line unicorn/import-style
+const exec = util.promisify(require('child_process').exec);
+
+/**
+ * Retrieves and parses the docker image metadata
+ * @see {@link https://github.com/docker/metadata-action}
+ */
+function getMetadata() {
+  try {
+    const raw = process.env.DOCKER_METADATA_OUTPUT_JSON;
```

Comment:
Do you have an example of what this looks like? I'm not sure I understand what labels and tags we might be applying here. (particularly the labels—I do see the tags in `release.yml`)

Reply:
Yep! Here's an example:

```json
{
  "tags": ["ghcr.io/readmeio/rdme:8.6.0-next.16", "ghcr.io/readmeio/rdme:next"],
  "labels": {
    "org.opencontainers.image.title": "rdme",
    "org.opencontainers.image.description": "ReadMe's official command-line interface (CLI) and GitHub Action 🌊",
    "org.opencontainers.image.url": "https://github.com/readmeio/rdme",
    "org.opencontainers.image.source": "https://github.com/readmeio/rdme",
    "org.opencontainers.image.version": "8.6.0-next.16",
    "org.opencontainers.image.created": "2023-03-21T21:10:25.208Z",
    "org.opencontainers.image.revision": "064e935860133f2fbe7ba0d6cb84507c65397689",
    "org.opencontainers.image.licenses": "MIT"
  }
}
```

I've created a gist that contains this and linked to it in the comments: https://github.com/readmeio/rdme/pull/777/commits/6b5c0c46498572375b56df9cdce5232693ccae69